src/lib.rs

```diff
@@ -394,8 +472,14 @@ impl KafkaJsonToDelta {
         &mut self,
         cancellation_token: Option<&CancellationToken>,
     ) -> Result<(), KafkaJsonToDeltaError> {
+        let dlq = dead_letters::dlq_from_opts(DeadLetterQueueOptions {
+            delta_table_uri: self.opts.dlq_table_location.clone(),
+        })
+        .await?;
```

Comment:
@mosyp and I discussed a few options offline. Presently, I'd like to leave as is tbh. We discussed lazy initialization of the dlq table as a possible solution, but tbh - I'd actually prefer eager loading and occupying memory for the dlq tx log immediately to allow for a fail fast on memory overhead rather than failing unexpectedly when the dlq is needed.

@mosyp - lmk if you disagree